src/engine/Game.java

```diff
@@ -86,7 +86,12 @@ private void playerTurn() {
 		List<Plant> plantsInGame = board.getPlantsInGame();
 		
 		for (Plant plant : plantsInGame) {
-			combat.plantAttack(plant);
+			if (!(plant instanceof Flower)) {
```

Comment:
We may need to bring back the separate "damage dealer" plant class assuming our further plant expansion will include other plants that don't do damage.